src/translations/en.json

```diff
@@ -32,6 +32,10 @@
       "on": "[%key:state::default::on%]"
     },
     "binary_sensor": {
+      "battery": {
```

Comment:
Put this below the `default` binary_sensor`.

Reply:
Good catch, done.